lib/__tests__/is-numeric.test.js

```diff
@@ -0,0 +1,21 @@
+const isNumeric = require('../is-numeric.js')
+const validNumbers = require('./__fixtures__/valid-numbers')
+const validNumericStrings = require('./__fixtures__/valid-numeric-strings')
```

Comment:
**This source code conforms to the [JavaScript Standard Code Style](https://standardjs.com/rules.html#semicolons)**. It is flagged as a violation/code smell/warning due to misconfigured (or simply neglected) settings.

I have since configured settings properly, and marking this as "resolved."